src/envoy/http/authn/http_filter_integration_test.cc

```diff
@@ -165,5 +165,35 @@ TEST_P(AuthenticationFilterIntegrationTest, CheckValidJwtPassAuthentication) {
   EXPECT_EQ("200", response->headers().Status()->value().getStringView());
 }
 
+TEST_P(AuthenticationFilterIntegrationTest, CORSPreflight) {
+  config_helper_.addFilter(kAuthnFilterWithJwt);
+  initialize();
+
+  // The AuthN filter requires JWT. The http request contains validated JWT and
+  // the authentication should succeed.
```

Comment:
I think the comment is not stale. `validated JWT`?

Reply:
Updated comments.